sdk/eventhub/Azure.Messaging.EventHubs/src/Processor/EventProcessor.cs

```diff
@@ -30,6 +29,21 @@ public class EventProcessor<T> where T : BasePartitionProcessor, new()
         /// <summary>The primitive for synchronizing access during start and close operations.</summary>
         private readonly SemaphoreSlim _runningTaskSemaphore = new SemaphoreSlim(1, 1);
```

Comment:
I've been converting `readonly` members back to the `const` style rather than the variable one that the formatter incorrectly chose.    That said, I've no issue if you leave them as-is for now.